### PR TITLE
Create security-policies.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Moved security policies to handbook from docs. Refer to: https://github.com/mattermost/docs/pull/3586.

### CODE REVIEW

**Review by TQuock — DISMISSED** [review]

File: operations/operations/company-policies/security-policies.md [hunk 1]
```diff
@@ -0,0 +1,427 @@
+1% Draft (WIP migration)
```
Comment (jasonblais) [inline operations/operations/company-policies/security-policies.md]: Do we need this line?

**Review by jasonblais — DISMISSED** [review]
One question, else LGTM

File: operations/operations/company-policies/security-policies.md (hunk 1 above)
Comment (justinegeffen) [reply operations/operations/company-policies/security-policies.md]: Good catch, thanks! We definitely don't.

**Comment by justinegeffen** [comment]
@jasonblais, @TQuock, I dismissed your reviews and it removed your approvals hence the re-request.

**Review by TQuock — APPROVED** [review]
approved to move this to another section of the handbook. We will review the contents through another PR

**Review by jasonblais — APPROVED** [review]

**Pull request merged**